android/app/src/main/res/values/strings.xml

```diff
@@ -194,4 +194,6 @@
     </string-array>
     <string name="label_miscellaneous_pref_category">Divers</string>
     <string name="label_shake_bug_reporter_invocation_method_pref">Secouer pour rapporter un bogue</string>
+    <string name="uri_heuristique" translatable="false">http://lheuristique.ca/</string>
```

Comment:
Can you move the url to this [const.xml](https://github.com/ApplETS/Notre-Dame/blob/master/android/app/src/main/res/values/const.xml)?

Reply:
Done !